src/pybricks/_common.pyi

```diff
@@ -110,7 +110,7 @@ class ColorLight:
 
 class LightArray:
     def __init__(self, n: int): ...
-    def on(self, brightness: int) -> None: ...
+    def on(self, brightness: (int, int, int, int)) -> None: ...
```

Comment:
Both a single value and a tuple are allowed here.

Reply:
Yes. However, since Python does not support method overloading, I did not know how to specify that.

This way, I've found that IntelliJ IDE does not complain about int/tuple arguments.

Reply:
There's the `Union` to allow multiple types. You can see various examples in `_common.pyi` for inspiration.

Reply:
Nice trick, thanks! Done.

Reply:
> Yes. However, since Python does not support method overloading, I did not know how to specify that.

FWIW, there is an [@overload](https://docs.python.org/3/library/typing.html#typing.overload) decorator.

But `Union` makes sense here.